docs/rules/paths-kebab-case.md

```diff
@@ -77,7 +77,7 @@ paths:
 
 - [path-excludes-patterns](./path-excludes-patterns.md)
 - [paths-segment-plural](./path-segment-plural.md)
-- [custom rules](./custom-rules.md)
+- [custom rules](./configurable-rules.md)
```

Comment:
Should these change everywhere?
```suggestion
- [configurable rules](./configurable-rules.md)
```

Reply:
Yes, but I can probably `sed` it, I did the links this way (which is how I didn't spot the words!)

Reply:
These are fixed now.